log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java

```diff
@@ -261,6 +281,12 @@ public void activateOptions() {
                 System.setProperty("java.security.krb5.conf", kerb5ConfPath);
             }
         }
+        if (saslMechanism != null) {
```

Comment:
looks like we need to consolidate with existing above code. What if user wants to set kerb5ConfPath and clientJaasConf?

Reply:
Then the user could set both. The block above isn't touching the `sasl.jaas.config` property. It lets you configure kerberos through system properties. I'm not sure why its so defensive but that seems orthogonal to this change.

Reply:
clientJaasConf can be used to  configure Kerberos. In that case, users may want to configure kerb5ConfPath. We may want  to pull  the below code to outside the if block..

```
            if (kerb5ConfPath != null) {
                System.setProperty("java.security.krb5.conf", kerb5ConfPath);
            }
```

Reply:
ok done